package.json

```diff
@@ -106,6 +107,13 @@
           ".arm"
         ]
       }
+    ],
+    "commands": [
```

Comment:
Hmm, when would you want users to utilize this command? Seems odd to expose it as part of the extension for others to utilize.

Reply:
Frankly it's there mostly for testing purposes.  I don't see that it causes any harm.